### PR TITLE
Clean up and fix translation tags

### PR DESCRIPTION
**Pull request by kluo**
@dcadams @caesar2164

### CODE REVIEW

File: lms/templates/courseware/course_about.html [hunk 1]
```diff
@@ -243,12 +243,14 @@
       <div class="inner-wrapper">
         ${get_course_about_section(course, "overview")}
       </div>
-      % if sneakpeek_allowed and not regularly_registered:
+      % if True:
```
Comment (dcadams) [inline lms/templates/courseware/course_about.html]: Just wondering why you've removed the sneakpeek condition.

Comment (caesar2164) [inline lms/templates/courseware/course_about.html]: looks like a testing change that slipped into the PR?

Comment (kluo) [inline lms/templates/courseware/course_about.html]: good catch let me change it back

File: cms/templates/keyword_sub_modal.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h3 class="title">
     % for keyword in keywords_supported:
       <tr>
         <td><input type="text" value="${keyword['name']}" readonly="readonly" /></td>
-        <td>${_(keyword['desc'])}</td>
+        <td>${keyword['desc']}</td>
```
Comment (stvstnfrd) [inline cms/templates/keyword_sub_modal.html]: Why aren't we translating this now?

Comment (kluo) [inline cms/templates/keyword_sub_modal.html]: It's getting translated in python now

**Pull request merged**